### DISCUSSION TITLE
Problem ketika mau daftar / buat akun baru

### DISCUSSION THREAD

**Discussion by TachooDa** [rizkyilhampra/spdhtc#78] [Q&A]
mao nanya lgi bro, kenapa pas mao daftar akun/create akun itu dia ngga bisa ya
![image](https://github.com/user-attachments/assets/348c1447-13ed-4a83-8662-6da0c892c451)

trus muncul error begini
Connection could not be established with host "mailhog:1025": stream_socket_client(): php_network_getaddresses: getaddrinfo for mailhog failed: No such host is known.

udah diubah mailhog nya ke local development pake 127.0.0.1 dia tetep ngga bisa

**Comment by rizkyilhampra** [comment]
Out of context menanyakan konfigurasi Mailhog disini, silahkan merujuk ke dokumentasi resminya atau tulisan lainnya terkait konfigurasinya. FYI Mailhog adalah salah satu mail driver untuk kebutuhan mail testing. Silahkan gunakan mail driver lain jika tidak bisa menggunakan Mailhog, lihat https://laravel.com/docs/9.x/mail#driver-prerequisites. 

Untuk default mailer per [v2.2-beta](https://github.com/rizkyilhampra/spdhtc/releases/tag/v2.2-beta) sudah saya ganti menjadi `log` dan juga telah ditambahkan driver baru `resend`(untuk *production*). Silahkan lihat perubahan pada `.env.example` untuk melihat konfigurasinya. 

Terima kasih 

-- Edit
Authentication termasuk Registration menggunakan Laravel Fortify, jika tidak ingin menggunakan Email Verification saat daftar akun, silahkan disable pada konfigurasi Fortify, selengkapnya lihat di dokumentasi Laravel.
